examples/TestEm3/TestEm3.cu

```diff
@@ -215,9 +224,9 @@ void TestEm3(const vecgeom::cxx::VPlacedVolume *world, int numParticles, double
   //  * objects to manage slots inside the memory,
   //  * queues of slots to remember active particle and those needing relocation,
   //  * a stream and an event for synchronization of kernels.
-  constexpr size_t TracksSize  = sizeof(Track) * Capacity;
-  constexpr size_t ManagerSize = sizeof(SlotManager);
-  const size_t QueueSize       = adept::MParray::SizeOfInstance(Capacity);
+  size_t TracksSize  = sizeof(Track) * Capacity;
+  size_t ManagerSize = sizeof(SlotManager);
+  size_t QueueSize   = adept::MParray::SizeOfInstance(Capacity);
```

Comment:
Why did you make these variables mutable? I do not see where you need to change them later.